snuba/datasets/discover.py

```diff
@@ -162,7 +162,7 @@ def __init__(self) -> None:
 
         self.__events_columns = ColumnSet(
             [
-                ("group_id", Nullable(UInt(64))),
+                ("group_id", UInt(64)),
```

Comment:
Let's leave it nullable. Once we will have the query AST we will be able to come back and fix this by tweaking the conditions instead.